MacDown/Code/Document/MPDocument.m

```diff
@@ -1363,6 +1345,32 @@ - (IBAction)render:(id)sender
 
 #pragma mark - Private
 
+- (void)toggleSplitterCollapseWithVisibility:(BOOL)isVisible
+                                 targetRatio:(CGFloat)targetRatio
```

Comment:
Or, as I just figured, it can take the panel as the sole parameter, and I think that would work.